api/serializers.py

```diff
@@ -23,13 +36,34 @@ class MediaSerializer(serializers.Serializer):
     ui_url = serializers.SerializerMethodField(
         help_text='A URL for the media item. This is a URL for the media UI, not a resource URL.'
     )
+    player_url = serializers.SerializerMethodField(
+        help_text='A URL to retrieve an embeddable player for the media item.'
+    )
+    sources = SourceSerializer(
+        help_text='A collection of download URLs for different media types.',
+        required=False, many=True
+    )
+    legacy_stats_url = serializers.SerializerMethodField(
```

Comment:
I think we should avoid having the legacy SMS mentioned explicitly. Instead, just have ``stats_ui_url``. Then when we add a native stats page, the API doesn't need to change.